src/seedu/addressbook/logic/Logic.java

```diff
@@ -85,7 +85,11 @@ public CommandResult execute(String userCommandText) throws Exception {
     private CommandResult execute(Command command) throws Exception {
```

Comment:
Remember to update your header comment, since `execute(Command command) `has a slightly modified behavior.